pyproject.toml

```diff
@@ -102,6 +102,8 @@ filterwarnings = [
   "ignore:.*Themes were changed to use evented model",  # specifying the full is not working here so glob pattern is used instead
   "ignore:distutils Version classes are deprecated::",
   "ignore:There is no current event loop:DeprecationWarning:",
+  # REMOVE me once we drop channel_axis.  Much easier to put here than to decorate every test.
+  "ignore:\\nThe 'channel_axis' argument is deprecated:DeprecationWarning",
```

Comment:
how many tests use this feature? It only postpones the job that needs to be done when removing these features as all these tests need to be fixed. So why not do this in this PR leaving only one test that verifies that it works.

Reply:
well, for one thing, it wasn't clear how we were going to fix it.  don't worry, the work will get done